static/docs/commands-reference/commit.md

```diff
@@ -162,22 +162,22 @@ DVC cache. But it is instructive to verify that's the case.
 First verification:
 
 ```dvc
-    $ dvc status
-
-    evaluate.dvc:
-        changed deps:
-            modified:           data/features
-            modified:           model.pkl
-    train.dvc:
-        changed outs:
-            not in cache:       model.pkl
+$ dvc status
+
+evaluate.dvc:
+    changed deps:
+        modified:           data/features
+        modified:           model.pkl
+train.dvc:
+    changed outs:
+        not in cache:       model.pkl
 ```
 
 And we can look in the DVC cache to see if the new version of `model.pkl` is
 indeed _not in cache_ as claimed. Look at `train.dvc` first:
 
 ```yaml
-    cmd: python src/train.py data/features model.pkl
+cmd: python src/train.py data/features model.pkl
     deps:
```

Comment:
move all the lines below to left

Reply:
@shcheklein Closing this pull request, will submit a new pull request with indentation changes only to DVC blocks.

Reply:
@ryokugyu no, don't do this

Reply:
@ryokugyu the change should be applied exactly the way you did - to all blocks

Reply:
this comment means that you should remove extra indentation for each line of this block, not the first one only

Reply:
removing all the extra spaces on other code blocks also.